lib/ansible/modules/system/filesystem.py

```diff
@@ -16,21 +16,22 @@
 author:
 - Alexander Bulimov (@abulimov)
 module: filesystem
-short_description: Makes file system on block device
+short_description: Makes filesystem
```

Comment:
i would rephrase: `Makes a file system` or `Makes file systems`

Reply:
I'm not sure which one is more valid `filesystem` or `file system` but in any case it should be either plural or have an 'a'

Reply:
> I'm not sure which one is more valid filesystem or file system

Used terms were not consistent (`filessytem`, `filesytem`, `file system`), I followed [man pages conventions](http://man7.org/linux/man-pages/man7/man-pages.7.html#STYLE_GUIDE) and choose `filesystem`.

> in any case it should be either plural or have an 'a'

Done